src/test/Makefile

```diff
@@ -157,6 +157,7 @@ OTHER_TESTS = \
 	checksum\
 	compat_incompat_features\
 	ctl_prefault\
+	doc_man\
```

Comment:
In my opinion it is "to late" to check files in generated dir.
Check probably should be added to md2man script.

Reply:
You're right — as mentioned in the pull request's description.  The plan was to have this simple test in 1.5 which doesn't move fast, and a real thing that generates docs from source in master.  But this plan didn't go well.